base/centos6/Dockerfile

```diff
@@ -8,12 +9,12 @@ RUN yum -y update && \
     bzip2-devel \
     dejavu-sans-fonts \
     lapack-devel \
-    libSM \
-    libXmu \
-    libXt \
     libcurl-devel \
     libicu-devel \
+    libSM \
     libtiff \
+    libXmu \
+    libXt \
```

Comment:
I had ordered these case-sensitively. Is case-insensitive ordering the best practice? I'm not opinionated, just curious.

Reply:
lol. Never ran into this on Ubuntu, so I have no personal opinion.